packages/warp-cache/src/cache.ts

```diff
@@ -406,6 +444,25 @@ export async function saveCache(
           reserveCacheResponse?.result?.gcs?.cache_key ?? ''
         )
         break
+
+      case 'azure_blob':
+        core.debug(`Saving Cache to Azure Blob`)
+        cacheKey = await cacheHttpClient.saveCache(
+          'azure_blob',
+          key,
+          cacheVersion,
+          archivePath,
+          // S3 Params are undefined for GCS
+          undefined,
+          undefined,
+          undefined,
+          undefined,
+          // GCS Params are undefined for Azure Blob
+          undefined,
+          undefined,
+          undefined,
+          reserveCacheResponse?.result?.azure_blob?.pre_signed_url ?? ''
```

Comment:
Why is this not an object?

Reply:
Not important to change now but this looks like a low hanging fruit to improve safety.

Reply:
Yup. This needs to be fixed.